README.md

```diff
@@ -399,15 +400,15 @@ in the `pom.xml`file. Otherwise, we would see this error:
       at io.grpc.internal.AbstractManagedChannelImplBuilder.<clinit>(AbstractManagedChannelImplBuilder.java:84)
       at uk.co.gresearch.spark.dgraph.connector.package$.toChannel(package.scala:113)
 
-Further, we need to set `protobuf-java >= 3.0.0` in the `pom.xml` file:
+Furthermore, we need to set `protobuf-java >= 3.0.0` in the `pom.xml` file:
 
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
       <version>[3,]</version>
     </dependency>
 
-to get rid of this error
+...to get rid of this error:
```

Comment:
Should there be a whitespace after the ellipsis? Can we use the ellipsis sign `…`?

Reply:
nope, fine with me, as long as you use the single character unicode ellipsis :-D